packages/shared/src/lib/cookie.ts

```diff
@@ -35,6 +35,53 @@ export const getCookieFeatureFlags = (): Record<string, FeatureValue> => {
   }
 };
 
+export const setCookie = (
+  name: string,
+  value: string | number | boolean,
+  options: Partial<{
+    domain: string;
+    expires: Date | string;
+    maxAge: number;
+    partitioned: boolean;
+    path: string;
+    sameSite: 'lax' | 'strict' | 'none';
+    secure: boolean;
+  }> = {},
+): void => {
```

Comment:
Just some non-blocking cleanups.
```suggestion
interface CookieOptions {
  domain: string;
  expires: Date | string;
  maxAge: number;
  partitioned: boolean;
  path: string;
  sameSite: 'lax' | 'strict' | 'none';
  secure: boolean;
}

export const setCookie = (
  name: string,
  value: string | number | boolean,
  options: Partial<CookieOptions> = {},
): void => {
```